roles/haproxy/templates/haproxy-dataplane.j2

```diff
@@ -0,0 +1,120 @@
+#---------------------------------------------------------------------
+# Example configuration for a possible web application.  See the
```

Comment:
Presumably this isn't an example config any more. Edit this comment to reflect reality.